docs/conf.py

```diff
@@ -111,6 +111,7 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
+remove_toctrees_from = ["demo/generated/*"]
```

Comment:
Is it the intention to use the extension in our demo docs? (I think that's OK, since we actually don't need those entries in the sidebar here, although of course it's not strictly showcasing the theme) 
But if so, I suppose you also need to add it in the `extensions` list?

Reply:
good point - I actually decided to just remove it from our docs because this is not strictly related to this theme and folks can look at the remove-toctrees repo for more information on this. Better to just show off our docs without extra extensions etc